### DISCUSSION TITLE
Publish a "pre-configured" `mkdocs-material` setup

### DISCUSSION THREAD

**Discussion by mo-lukecarr** [squidfunk/mkdocs-material#7009] [Q&A]
Is there a way to effectively publish a pre-configured setup for `mkdocs-material`? That is, effectively a theme of the theme?

My context is that, over time, I've managed to create a configuration of `mkdocs-material` (`mkdocs.yml` and a collection of extra stylesheets/JS) that aligns with internal brand guidelines for my org. Is there a way that I can effectively package this so that my colleagues can install my package (rather than `mkdocs-material` directly) and get all of this theming out-of-the-box?

Specific pain points that I imagine: if I'm providing a pre-configured `mkdocs.yml`, how would my colleagues bring their own configuration for MkDocs? Would their file override any values that I've defined?

**Comment by alexvoss** [comment]
Hi, this discussion here https://github.com/squidfunk/mkdocs-material/discussions/6994 might help. For the question about the `mkdocs.yml` you may want to look at [configuration inheritance](https://www.mkdocs.org/user-guide/configuration/#configuration-inheritance)? Hope this helps.
